free-2.md

```diff
@@ -46,6 +53,9 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+In 2014, NCarter3 created a similar DSL for CS111 [here](https://github.com/NCarter3/project/blob/master/documents/description.md). He attempted to "create a new DSL that wraps Todo.txt with powerful calendar calculations. Effectively, I want it to tell me what to do next and when to do it". His project focused on making a calendar more like a todo list which could tell him what activity he should do next. 
+
+In 2015, aputman created a project along these lines [here](https://github.com/aputman/project/blob/master/documents/final.md) for CS111 as well. His idea was to focus on making it easier for users to scope "different events with different dates" for things like classes which meet several times during every week except during breaks and holidays or finals week when the schedule is different. I got my idea before I began researching but when I found these I liked them both and have borrowed ideas from both in this write up.
```

Comment:
+1 If you choose this idea, feel free to talk to me about challenges, limitations, etc.